python/tk_syntheyes/ui/main_window.py

```diff
@@ -181,6 +184,38 @@ def _init_panel(self, panel_type: type, name: str, parent_panel: QWidget, visibl
 
         return panel
 
+    @property
+    def inbuilt_apps(self):
+        if not getattr(self, "_inbuilt_apps", None):
+            self._inbuilt_apps = {}
+            inbuilt_apps_path = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir, "inbuilt_apps"))
+            if os.path.isdir(inbuilt_apps_path):
+                import importlib
+                import inspect
+                for file in os.listdir(inbuilt_apps_path):
+                    file_path = os.path.join(inbuilt_apps_path, file)
+                    if not os.path.isfile(file_path): continue
+
+                    spec = importlib.util.spec_from_file_location(file.rsplit(".", 1)[0], file_path)
+                    if not spec: continue
+
+                    self._engine.log_debug("Loading inbuilt app: %s", file)
+                    mod = importlib.util.module_from_spec(spec)
+                    spec.loader.exec_module(mod)
+                    if not mod: continue
+
+                    # iterate over all classes
+                    for cls_name, cls in inspect.getmembers(mod, inspect.isclass):
+                        if cls != InbuiltApp and issubclass(cls, InbuiltApp):
+                            self._inbuilt_apps[cls_name] = cls(self._engine)
+            
+        return self._inbuilt_apps
```

Comment:
- Das hier hat relativ viele abstufungen. Geht das noch irgendwie schöner und weniger tief?
- in code imports wenn es geht vermeiden. Imports sollten immer am anfang eines modules stehen.
- formatierung issues bei den if not x: continues (ja ich weiß beißt sich mit Feedback 1 ;) aber trotzdem)